frontend/src/app/okrview/objective/objective-contents/objective-contents.component.html

```diff
@@ -30,9 +30,7 @@
     </div>
   </div>
   <div *ngIf="currentUserRole.isAtleastOKRManager() && !cycle.isCycleClosed()" class="f-1">
-    <button mat-mini-fab color="accent" class="add-keyresult-button" (click)="clickedAddKeyResult()" [disabled]="objective.keyResultIdList.length >= maxKeyResults">
-      <mat-icon matTooltip="{{ 'objective-contents.tooltip.define-new-key-result' | translate }}">add_circle_outline</mat-icon>
-    </button>
+    <app-add-button class="add-button" text="{{'objective-contents.define-new-key-result' | translate}}" (buttonClicked)="clickedAddKeyResult()" [isDisabled]="objective.keyResultIdList.length >= maxKeyResults"></app-add-button>
```

Comment:
wieder:
Wieso hat der add-button nochmal die Klasse "Add-button"

Reply:
der Klassenname sollte so Aussagekräftig sein, dass er das beschreibt, was die Klasse auch tut.
Also in dem Fall etwas wie "position-left" oder "leftsided"

Reply:
ansonsten gibt es auch schon die klassen "left" und "right" in der _core.scss, vielleicht erfüllen diese auch schon den Zweck